docs/development/workflow/get_devel_version.rst

```diff
@@ -166,10 +166,8 @@ show you all of the remotes it knows about for your local repository of
 Note that `git`_ already knew about one remote, called *origin*; that is your
 fork of `Astropy`_ on `GitHub`_.
 
-To make more explicit that origin is really *your* fork of `Astropy`_, rename that
-remote to your `GitHub`_ user name::
-
-  git remote rename origin your-user-name
```

Comment:
Very much agreed, otherwise nothing can be copy-pasted anymore. (Personally, I also keep `upstream` rather than `astropy`, but I think `astropy` is fine too.)

Reply:
I agree with this change, but we should be careful to update all of the developer documentation. For example the following should also be updated, and there might be more: https://github.com/astropy/astropy/blob/8a63fa745ce1cb59fcdafd6742cac28847f87281/docs/development/workflow/development_workflow.rst#L62-L65

Reply:
I tried to grep for it and replace but must have missed some variants. I will replace this one too.

Reply:
@mhvk , I was tempted to replace `astropy` with `upstream` but that would blow up the diff even more, so I didn't.

Reply:
Changing the name of the user's fork from their username to `origin` is a good change because it makes it possible to copy-paste more examples. Changing `astropy` to `upstream` does not make it easier to copy-paste, so I don't mind not making that change.